### DISCUSSION TITLE
How configure Nginx Proxy Manager?

### DISCUSSION THREAD

**Discussion by gatodelcaribe** [plausible/analytics#4550] [Self-Hosted Support]
I tried to follow the Nginx steps into GitHub but it doesn't work if I want to configure Nginx Proxy Manager, is there any way to do it? On NPM 
<img width="499" alt="Screenshot 2024-09-09 at 05 28 55" src="https://github.com/user-attachments/assets/a0ab6580-1495-48c2-8f9d-b7d3e9786a9c">

On local as Localhost:8000 it works perfectly.

**Comment by ruslandoga** [comment]
👋 @gatodelcaribe 

We only have the guide for plain Nginx configuration: https://github.com/plausible/community-edition/tree/v2.1.1/reverse-proxy#nginx

If you figure out how to make Nginx Proxy Manager work, please consider writing a guide and we will link to it from documentation! :)
